pkg/ingester/stream.go

```diff
@@ -342,6 +350,18 @@ func (s *stream) storeEntries(ctx context.Context, entries []logproto.Entry, usa
 			}
 			continue
 		}
+		if dup {
+			if s.configs != nil {
+				if s.configs.LogDuplicateMetrics(s.tenant) {
+					s.metrics.duplicateLogBytesTotal.WithLabelValues(s.tenant).Add(float64(len(entries[i].Line)))
```

Comment:
this PR adds two CLI configs, one for the metric and one for the logs. WDYT of making both working behind a single one? Asking because I can't think of a scenario were I'd want only one of them. But having both means another CLI flag to Loki.

Reply:
@slim-bean Argues that we need two, which is why we added it